packages/builder/src/builtin-tasks/console.ts

```diff
@@ -15,36 +16,39 @@ export default function (): void {
     .setAction(
       async (
         { noCompile }: { noCompile: boolean }, // eslint-disable-line
-        { config, run, runtimeArgs }           // eslint-disable-line
-
+        runtimeEnv: AlgobRuntimeEnv            // eslint-disable-line
       ) => {
-        if (config.paths == null) {
+        if (!runtimeEnv.config.paths) {
           return;
         }
+        const paths = runtimeEnv.config.paths;
 
         // if (!noCompile) {
         //  await run("compile");
         // }
 
-        await fsExtra.ensureDir(config.paths.cache);
-        const historyFile = path.join(
-          config.paths.cache,
-          "console-history.txt"
-        );
+        await fsExtra.ensureDir(paths.cache);
+        // const historyFile = path.join(
+        //  paths.cache,
+        //  "console-history.txt"
+        // );
 
-        const nodeArgs = [];
-        if (semver.gte(process.version, "10.0.0")) {
-          nodeArgs.push("--experimental-repl-await");
-        }
+        // const nodeArgs = [];
+        // if (semver.gte(process.version, "10.0.0")) {
+        //  nodeArgs.push("--experimental-repl-await");
+        // }
```

Comment:
Why this code is commented?
Please use `/*...*/` for commenting out multiple lines of code.
Maybe it's worth to add a note if we need this code or not.

Reply:
Linter wasn't happy but I thought I should keep the code.
What do you think?

Reply:
Not sure if we need it (we can always roll-back). but for commenting section of code we should use `/*...*/` because otherwise we need to think twice if there are other changes or not.

Reply:
I think about removing `console.ts` completely

Reply:
I don't think we should spend much time on it now. If it passes tests and compiles then let's leave.